### PR TITLE
Remove contribution tool

### LINKED ISSUE #93: Extract contribution UI from public website
The contribution app should be standalone. This will decrease the complexity of the website and enable easier maintenance of the Contribution UI.

- [x] Deploy on `contribute.opentermsarchive.org`.
- [x] Ensure the documentation is up to date.
- [x] Ensure existing links in github issues still work (redirection)

### PR DESCRIPTION
**Pull request by martinratinaud**
Fixes #93

### CODE REVIEW

**Review by clementbiron — APPROVED** [review]
I'm not abble to review the ops modifications.
I tried the project localy and everything looks good.

File: ops/roles/ota/github-clone/tasks/main.yml [hunk 1]
```diff
@@ -24,15 +27,15 @@
       pull: yes
       args:
         ENV_FILE: '{{ env_file }}'
-      path: '/home/{{ ansible_user }}/{{ app }}'
-    name: '{{ app }}:{{ result_git_clone.before }}'
+      path: '/home/{{ ansible_user }}/{{ name }}'
+    name: '{{ name }}:{{ git_clone_result.after[:7] }}'
```
Comment (Ndpnt) [inline ops/roles/ota/github-clone/tasks/main.yml:31]: What is `after[:7]`?

Comment (martinratinaud) [reply ops/roles/ota/github-clone/tasks/main.yml:31]: This is to keep only the first 7 characters of the commit SHA

Comment (Ndpnt) [reply ops/roles/ota/github-clone/tasks/main.yml:31]: Okay thanks, maybe add a comment

Comment (Ndpnt) [reply ops/roles/ota/github-clone/tasks/main.yml:31]: Sorry I did not see it was already merged

**Pull request merged**